CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+
+Open the installer and let it run. Then "Clone a repository from the Internet", which can also be reached from `File > Clone repository...` or `ctrl + shift + o`. Go to the third tab and paste in the link to your cloned branch and choose an empty local path, then press 'Clone'.
+
+From here you can open it in your file explorer or preferred code editor via the buttons or manually in those programs
+
+### Writing in markdown 
+[markdownguide.org](https://www.markdownguide.org) has a wonderful [cheat sheet](https://www.markdownguide.org/cheat-sheet/) for writing in markdown. All of the documentation is written in markdown so knowing this is crucial. If you would like to see examples of how to format your documentation go to the documentation we have already written and look at the raw markdown. The most commonly used syntax are # to indicate titles and subtitles so be sure to get familiar with that.
+
+### Testing your version of the site
+The site is written in [vitepress](https://vitepress.dev), which is a simlpe way to write documentation. They have a simple [getting started guide](https://vitepress.dev/guide/getting-started) which will help you install and use vitepress.
+
+To summarize what you most likely need to know:
+- Open the console for your operating system and run the command `npm install -D vitepress`. The [getting started guide](https://vitepress.dev/guide/getting-started) has alturnatives to npm if your system does not recognize it as a command
+- Use the `cd` command to navigate to the install location of our wiki's root folder
+- Use the command `npm run docs:dev` to start the site at http://localhost:5174/  
+If you run into an error, read it and try to understand what's wrong. If you cannot find a solution consult the [vitepress documentation](https://vitepress.dev/guide/getting-started)
+
+### Adding to documentation
+In the `/docs/` folder you will find many files.  
+You can ignore any `index.md` files, and any `.vitepress` files.
+
+From here navigate to the documentation you would like to add to and add what you would like to. Try and keep in line with the style already used in other documentation.
+
+Once you have made your changes [test how the markdown will look](https://markdownlivepreview.com) and look at the changes on a local host of the site. 
+
+Finally, back on Github Desktop make sure the file(s) you have changed are ticked in the changes tab on your Wiki branch repository, give them a summary, and then commit them. 
+
+### Adding new documentation
+
+#### New page in a present topic
+Adding a new page is more complex than adding to a page that is already there. First, navigate to the file where you would like to make a new page that corrosponds to the location on the site. For example if you wanted to write about an NCEA standard go to `docs/ncea/` and then the level it is for. There, create a new markdown file named `your documentation name.md`.
```

Comment:
We should add a note saying it should be lowercase and also no spaces/numbers ([a-z_]+\.md)